zebrad/tests/common/get_block_template_rpcs/get_block_template.rs

```diff
@@ -27,6 +27,10 @@ use crate::common::{
 /// We've seen it take anywhere from 1-45 seconds for the mempool to have some transactions in it.
 pub const EXPECTED_MEMPOOL_TRANSACTION_TIME: Duration = Duration::from_secs(45);
 
+/// Number of times we want to try submitting a block template as a block proposal at an interval
+/// that allows testing the varying mempool contents.
+const NUM_SUCCESSFUL_BLOCK_PROPOSALS_REQUIRED: usize = 3;
```

Comment:
```suggestion
const NUM_SUCCESSFUL_BLOCK_PROPOSALS_REQUIRED: usize = 10;
```

Reply:
This will test 40 time variants of 10 templates, and take around 8 minutes.

Since the whole test suite takes around 2 hours, I think that's ok.